packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3513,10 +3513,20 @@ export default class Meeting extends StatelessWebexPlugin {
    * @returns {Promise<void>} resolves when the brb status is updated or does nothing if not in a multistream meeting.
    * @throws {Error} - Throws an error if the request fails.
    */
-  // eslint-disable-next-line consistent-return
   public async beRightBack(enabled: boolean): Promise<void> {
     // this logic should be applied only to multistream meetings
+    if (!this.isMultistream) {
+      LoggerProxy.logger.error('Meeting:index#beRightBack --> Not a multistream meeting');
+    }
+    if (!this.mediaProperties.webrtcMediaConnection) {
+      LoggerProxy.logger.error(
+        'Meeting:index#beRightBack --> WebRTC media connection is not defined'
+      );
+
+      return;
```

Comment:
Why do we need to return here? Shouldn't `LoggerProxy.logger.error` throw and therefore we don't need a return?

Reply:
`LoggerProxy.logger.error` doesn't throw an error, it uses `console.error` to print the error but doesn't throw it.

Reply:
Double checked this, it seems like you're right, @brycetham. Thank you for checking.